gradle-module/src/main/java/org/example/Main.java

```diff
@@ -22,4 +23,11 @@ public static void main(String[] args) {
     private static void test() {
         System.out.println("test");
     }
-}
\ No newline at end of file
+
+   /* asd
+    we
+    asd
+
+    */
+    
```

Comment:
[Category: Maintainability] [Severity: Low] The comment block at the end of the file is not properly formatted and contains placeholder text. This can lead to confusion and should be removed or replaced with meaningful documentation. Rationale: Clear and meaningful comments improve code maintainability and understanding. [Reference: Best practices for commenting code]